app/models/user.rb

```diff
@@ -19,7 +19,7 @@ class User < ApplicationRecord
   include BackwardsCompatiblePasswordDecryptor
 
   # Allows other models to get the current user with User.current_user
-  cattr_accessor :current_user
+  thread_cattr_accessor :current_user
```

Comment:
This has no effect in production because we have puma set to only use on thread. However, I think it may help in tests. Maybe. Possibly. It shouldn't hurt, at least.